contracts/cw-ics721-bridge/src/contract.rs

```diff
@@ -79,48 +77,19 @@ fn execute_callback(
         Err(ContractError::Unauthorized {})
     } else {
         match msg {
+            CallbackMsg::CreateVouchers { receiver, create } => {
```

Comment:
Requesting diagram to show how all these vouchers interact with each other

Reply:
![image](https://user-images.githubusercontent.com/30676292/210026430-ab673969-23b7-4ffd-964c-d22453e5adeb.png)

wdyt of this?